harvester/cli.py

```diff
@@ -32,3 +35,85 @@ def shell(ctx: click.Context) -> None:
     """Run a bash shell inside the docker container."""
     # ruff: noqa: S605, S607
     os.system("bash")
+
+
+@main.command()
+@click.option(
```

Comment:
Thanks for bringing this up again.  I agree that a default would be ideal.  During testing, it can be handy to have the ability to name the crawl as this will show up as a folder.  But, given that you can also name the output files, no need for the crawl name outside of testing most likely.